src/OpenSSL/SSL.py

```diff
@@ -1,3 +1,19 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2001 Martin Sjögren and pyOpenSSL contributors
```

Comment:
Since I wrote all of the code in this module from scratch (but not clean-room), I'm not sure Martin's name belongs here.  Same comment for most of the other Python implementation and test modules.